src/fastlane/Fastfile

```
@@ -0,0 +1,315 @@
+        description: "[View Run on GitHub](" + run_url + ")",
+        color: color,
+        fields: [
+          { name: "Branch", value: branch, inline: true },
+          { name: "Commit", value: "`#{commit[0..7]}`", inline: true }
+        ]
+      }
+    ]
+  }
+
+  sh("curl -X POST -H 'Content-Type: application/json' -d '#{message.to_json}' #{webhook_url}")
+end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `notify_discord` is too high. [<8, 21, 5> 23.02/23]

(Metrics/AbcSize)